src/molecule_plugins/podman/playbooks/create.yml

```diff
@@ -11,9 +11,7 @@
     - name: Log into a container registry
       containers.podman.podman_login:
         certdir: >-
-          {% if lookup('env', 'DOCKER_CERT_PATH') %}
-          {{ item.cert_path | default(lookup('env', 'DOCKER_CERT_PATH') + '/cert.pem') }}
-          {% endif %}
+          {% if lookup('env', 'DOCKER_CERT_PATH') %}{{ item.cert_path | default(lookup('env', 'DOCKER_CERT_PATH') + '/cert.pem') }}{% else %}{{ item.cert_path | default(omit) }}{% endif %} # noqa yaml[line-length] jinja[spacing]
```

Comment:
Did you try with ``noqa:`` instead of ``noqa`` ?

Reply:
btw, why do you need to ignore the line-length ? I've not checked but the line is using '>-' so you should be able to split over the line the same way it was before your change. It would be easier to read and no error from the lint.

Side note: it would be interesting to check if it's possible to reproduce that linting issue and ask -lint developers as it would be surprising to not be able to ignore an error.